{{cookiecutter.project_slug}}/{{cookiecutter.project_slug}}/routes/endpoints/item.py

```diff
@@ -7,59 +7,52 @@
 from fastapi import APIRouter
 from fastapi.responses import JSONResponse
 
-import {{cookiecutter.project_slug}}.schemas.general as general_schema
-import {{cookiecutter.project_slug}}.schemas.item as item_schema
+import {{ cookiecutter.project_slug }}.schemas.general as general_schema
+import {{ cookiecutter.project_slug }}.schemas.item as item_schema
 
 router = APIRouter()
 
-TAG = ['Item']
+TAG = ["Item"]
 
 data = [
     {"id": "1", "price": 12.99},
     {"id": "2", "price": 15.89},
     {"id": "3", "price": 11.19},
-    {"id": "4", "price": 13.49}
+    {"id": "4", "price": 13.49},
```

Comment:
Why a comma in the last element of a list?

Reply:
That's also my question because I realized that testing lint by ruff complains it!
I had to add comma to resolve it.
If I remove comma and run `pre-commit run --all-files` I get:
```
--- fastapi_boilerplate/routes/endpoints/item.py
+++ fastapi_boilerplate/routes/endpoints/item.py
@@ -17,7 +17,7 @@
     {"id": "1", "price": 12.99},
     {"id": "2", "price": 15.89},
     {"id": "3", "price": 11.19},
-    {"id": "4", "price": 13.49}
+    {"id": "4", "price": 13.49},
 ]
```
in output.